### PR TITLE
chore: Mzk/readme installation

### PR DESCRIPTION
**Pull request by mzkrasner**
1. Moves contributing, packaging, and releasing into a separate CONTRIBUTING.md file
2. Includes installation guide for mac and linux
3. Includes docker guides

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,49 +15,101 @@ For example, to enable debug logging for code from this repo but error logging f
 
     $ RUST_LOG=ERROR,ceramic_kubo_rpc=DEBUG,ceramic_one=DEBUG cargo run -p ceramic-one -- daemon
 
-## Packaging
-To package rust-ceramic, you will need the following dependencies
+## Installation
```
Comment (oed) [inline README.md:18]: Installation should go above usage?

Also, once installed I assume usage doesn't need to include cargo?

**Review by oed — COMMENTED** [review]
Thanks for the quick fix @mzkrasner 

I assume the **Generating Servers** section should also go into contributing? 

Should we also link the contributing file from the main readme?

File: README.md (hunk 1 above)
Comment (dav1do) [reply README.md:7]: moved usage below installation and updated to use the binary directly. also added some info about available flags

**Comment by mzkrasner** [comment]
@oed @dav1do I also moved "generating servers" section to contributing doc

**Review by oed — APPROVED** [review]
Great, lgtm. Thanks to you both!

**Pull request merged**